starter-code/css/main.css

```diff
@@ -6,3 +6,220 @@ Green: #00B172
 White: #FFF
 
 */
+
+
+/*****************************************************************
+        RESETS
+*****************************************************************/
+
+
+html, body, div, span, applet, object, iframe,
```

Comment:
Nice ! 

It's good to reset the default style of the browser before building your website :)